tests/unit/helper.py

```diff
@@ -79,12 +79,8 @@ def create_instance_of_described_class(self):
         If cls.example_data is None, just create a simple instance of the
         class.
         """
-        if self.example_data:
-            instance = self.described_class(self.example_data,
-                                            self.session)
-        else:
-            instance = self.described_class()
-            instance.session = self.session
+        instance = self.described_class(self.example_data)
```

Comment:
You mean the portion that does

```py
if self.example_data
    instance - self.described_class(self.example_data, self.session)
```

That is absolutely used in numerous places. We don't call this method explicitly but that's why its used in other methods on this class and relied upon implicitly.

Hint, the second part of the `if/else` statement is also used in at least one place.

Can you tell me *exactly* what was going wrong with this as it was originally? I'll help you fix it. @ueg1990 is a new contributor who means well but does not represent the project and is not as familiar with it as @itsmemattchung and I are.